src/visualizations/config/adapters/dhis_highcharts/subtitle/index.js

```diff
@@ -59,6 +60,9 @@ export default function (series, layout, metaData, dashboard) {
                     Boolean(!dashboard)
                 )
                 break
+            case VIS_TYPE_SCATTER:
+                subtitle.text = dashboard || filterTitle
```

Comment:
Not sure if this is a safe way to do the `dashboard` check (returns `text: true`?), but it's already in use on line 33 so I assume it's ok.